package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
What I might do instead is properly add highland as a dev dependency to cleverjs. That would make the other PRs less confusing